### ISSUE TITLE
Rank Error?

### ISSUE THREAD

**Issue by mayork** [hoburg/gpkit#729]
I'm getting the following error in https://github.com/hoburg/gpkit-models/blob/commercial_sizing/gpkitmodels/Commercial%20Sizing/Commerical_Flight_Profile.py

Traceback (most recent call last):
  File "C:\Users\mayork\Documents\GitHub\gpkit-models\gpkitmodels\Commercial Sizing\Commerical_Flight_Profile.py", line 263, in <module>
    CommericalAircraft()
  File "C:\Users\mayork\Documents\GitHub\gpkit-models\gpkitmodels\Commercial Sizing\Commerical_Flight_Profile.py", line 260, in __init__
    sol = m.solve()
  File "C:\Python27\lib\site-packages\gpkit\constraints\prog_factories.py", line 80, in solvefn
    result = solvefn(solver, verbosity-1, *args, **kwargs)
  File "C:\Python27\lib\site-packages\gpkit\geometric_program.py", line 171, in solve
    k=self.k, *args, **kwargs)
  File "C:\Python27\lib\site-packages\gpkit\_cvxopt.py", line 45, in cvxoptimize
    solution = gpsolver(k, F, g)
  File "C:\Python27\lib\site-packages\cvxopt\cvxprog.py", line 2162, in gp
    return cp(Fgp, G, h, dims, A, b)
  File "C:\Python27\lib\site-packages\cvxopt\cvxprog.py", line 1969, in cp
    xdot_e, xaxpy_e, xscal_e)
  File "C:\Python27\lib\site-packages\cvxopt\cvxprog.py", line 788, in cpl
    raise ValueError("Rank(A) < p or "\
ValueError: Rank(A) < p or Rank([H(x); A; Df(x); G]) < n

the line causing it is 

dhft[iclimb1] == tmin[iclimb1] * RC[iclimb1]

 All the variables are VectorVariables. I checked and all the variables in the constraint have a length of 2. What else can cause this error?

**Comment by bqpd** [comment]
Hmm, sometimes cvxopt can't solve certain problems that MOSEK can, and I
have a feeling this might be one of them... @whoburg?

**Comment by whoburg** [comment]
yeah this often to happen when there's an unbounded variable or otherwise poorly formulated model (e.g. if a variable is constrained on both slides but nonetheless wants to slide off to infinity due to the pressures on it).

Does mosek solve the model?

**Comment by bqpd** [comment]
@mayork, did this get resolved?

**Comment by mayork** [comment]
@bqpd I think its a cvxopt problem. I kept working on the model and eventually it solved, but I added to it more and now it does the thing where the solver appears to be bouncing back and fourth between two suboptimal points. I'm going to install Mosek this week and give it a go with that

Side note - for a while I was confused and didn't realize the rank error is basically the same as infeasible, so I've started a list of all the errors I encounter and where to look when debugging them. I was hoping to make a section in the documentation about this because I currently don't see one
